### PR TITLE
[Edn/tr-tr]

### PR DESCRIPTION
**Pull request by LeaveNhA**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [ ] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [ ] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: tr-tr/edn-tr.html.markdown [hunk 1]
```diff
@@ -0,0 +1,156 @@
+---
+language: edn
+filename: learnedn.edn
```
Comment (divayprakash) [inline tr-tr/edn-tr.html.markdown]: Please fix this to -
```
filename: learnedn-tr.edn
```

File: tr-tr/edn-tr.html.markdown [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+contributors:
+  - ["Seçkin KÜKRER", "https://github.com/LeaveNhA"]
+---
```
Comment (divayprakash) [inline tr-tr/edn-tr.html.markdown]: Please add the contributors field from the original English doc here and add your name in the translators section. Additionally, please add the following lang line -
```
lang: tr-tr
```

Comment (LeaveNhA) [reply tr-tr/edn-tr.html.markdown]: Firstly, this is an independent documentation written by me.
Nothing was used in it.
Still required to add original document's contributors ?

Comment (LeaveNhA) [reply tr-tr/edn-tr.html.markdown]: And, this document is not a `Translation`, It's original btw.

File: tr-tr/edn-tr.html.markdown (hunk 1 above)
Comment (LeaveNhA) [reply tr-tr/edn-tr.html.markdown]: Fixed.

File: tr-tr/edn-tr.html.markdown (hunk 2 above)
Comment (LeaveNhA) [reply tr-tr/edn-tr.html.markdown]: Fixed!

Comment (divayprakash) [reply tr-tr/edn-tr.html.markdown]: > Firstly, this is an independent documentation written by me.
> Nothing was used in it.
> Still required to add original document's contributors ?

My apologies, you can remove the original document's contributors

Comment (divayprakash) [reply tr-tr/edn-tr.html.markdown]: > And, this document is not a `Translation`, It's original btw.

Please put this somewhere in the PR next time.

Comment (divayprakash) [reply tr-tr/edn-tr.html.markdown]: > Additionally, please add the following lang line -
> 
> ```
> lang: tr-tr
> ```

This change still needs to be made.

Comment (LeaveNhA) [reply tr-tr/edn-tr.html.markdown]: > My apologies, you can remove the original document's contributors
No problem.

> This change still needs to be made.
It's done! 👨🏻‍💻

**Pull request merged**